hangman/lib/hangman/connector.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Hangman.Connector do
+  def child_spec(_arg) do
```

Comment:
Does this need to be a process? It's stateless.

Reply:
I designed the connector as an agent between a genserver and other modules. The connector itself is not a process I think... It is just designed as a module to help the supervisor or the controller create and then operate a genserver and allow us to put some customized  configurations if needed in future.